modules/text_server_adv/text_server_adv.cpp

```diff
@@ -1598,14 +1594,11 @@ _FORCE_INLINE_ bool TextServerAdvanced::_ensure_cache_for_size(FontDataAdvanced
 			FT_Done_MM_Var(library, amaster);
 		}
 #else
-		ERR_FAIL_V_MSG(false, TTR("FreeType: Can't load dynamic font, engine is compiled without FreeType support!");
+		ERR_FAIL_V_MSG(false, RTR("FreeType: Can't load dynamic font, engine is compiled without FreeType support!");
 #endif
 	} else {
 		// Init bitmap font.
 		fd->hb_handle = hb_bmp_font_create(fd, nullptr);
```

Comment:
It is probably a good idea not to use `hb_` prefix for your functions, to avoid potentially clashing with HarfBuzz API if it ever provided a function with the same name.